homeassistant/components/firmata/light.py

```diff
@@ -0,0 +1,101 @@
+"""Support for Firmata light output."""
+
+import logging
+from typing import Type
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    SUPPORT_BRIGHTNESS,
+    LightEntity,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_MAXIMUM, CONF_MINIMUM, CONF_NAME, CONF_PIN
+from homeassistant.core import HomeAssistant
+
+from .board import FirmataPinType
+from .const import CONF_INITIAL_STATE, CONF_PIN_MODE, DOMAIN
+from .entity import FirmataPinEntity
+from .pin import FirmataBoardPin, FirmataPinUsedException, FirmataPWMOutput
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, config_entry: ConfigEntry, async_add_entities
+) -> None:
+    """Set up the Firmata lights."""
+    new_entities = []
+
+    board = hass.data[DOMAIN][config_entry.entry_id]
+    for light in board.lights:
+        pin = light[CONF_PIN]
+        pin_mode = light[CONF_PIN_MODE]
+        initial = light[CONF_INITIAL_STATE]
+        minimum = light[CONF_MINIMUM]
+        maximum = light[CONF_MAXIMUM]
+        api = FirmataPWMOutput(board, pin, pin_mode, initial, minimum, maximum)
+        try:
+            api.setup()
+        except FirmataPinUsedException:
+            _LOGGER.error(
+                "Could not setup light on pin %s since pin already in use.",
```

Comment:
We don't end logging messages with period.

Reply:
I will update this in all of the component.